test/GlobStream.js

```diff
@@ -0,0 +1,58 @@
+var stream = require('../stream');
+var expect = require('expect');
+var miss = require('mississippi');
+
+var concat = miss.concat;
+var pipe = miss.pipe;
+
+describe('GlobStream', function() {
+  it('emits an error if it is not first in the stream', function(done) {
+    var gs1 = stream('./fixtures/whatsgoingon/**/*.txt', [], { cwd: __dirname, });
+    var gs2 = stream('./fixtures/whatsgoingon/**/*.txt', [], { cwd: __dirname, });
+
+    pipe([
+      gs1,
+      gs2,
+    ]);
+  });
+
+  it('emits an error if there are no matches', function(done) {
+    var gs = stream(
+      './fixtures/whatsgoingon/kojaslkjas.txt',
+      [],
+      { cwd: __dirname, }
+    );
+
+    gs.once('error', function(err) {
```

Comment:
This should probably watch for end of stream instead of just on error.  I think there's a utility in mississippi